official/marketplace/tools.go

```diff
@@ -0,0 +1,10 @@
+//go:build tools
```

Comment:
I think we can use only one tools file

https://github.com/ignite/apps/blob/main/integration/tools.go